ntc_templates/templates/fsas_sir_show_ip_route.textfsm

```diff
@@ -0,0 +1,15 @@
+Value FLAG ([\*X])
+Value PROTOCOL (\w+)
+Value NETWORK (\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})
+Value PREFIX_LENGTH (\d{1,2})
+Value GATEWAY (\S+)
+Value DISTANCE (\d+)
+Value UPTIME (\S+)
+Value INTERFACE (\w+)
+
+Start
+  ^FP\s+Destination\/Mask
+  ^[-\s]+$$
+  ^(\s|${FLAG})${PROTOCOL}\s+${NETWORK}\/${PREFIX_LENGTH}\s+${GATEWAY}\s+${DISTANCE}\s+${UPTIME}\s+${INTERFACE}\s*$$ -> Record
```

Comment:
We can simplify some regexes (with no apparent issues) to hopefully future-proof.
If you're open to these suggestions, please click the "commit suggestions" button to accept them. Thank you!

```suggestion
Value FLAG (\S)
Value PROTOCOL (\w+)
Value NETWORK (\S+)
Value PREFIX_LENGTH (\d+)
Value GATEWAY (\S+)
Value DISTANCE (\d+)
Value UPTIME (\S+)
Value INTERFACE (\w+)

Start
  ^FP\s+Destination\/Mask
  ^---\s+
  ^\s*${FLAG}?${PROTOCOL}\s+${NETWORK}\/${PREFIX_LENGTH}\s+${GATEWAY}\s+${DISTANCE}\s+${UPTIME}\s+${INTERFACE}\s*$$ -> Record
```